### ISSUE TITLE
[Dev UI] More metadata in Eval run table

### ISSUE THREAD

**Issue by ssbushi** [firebase/genkit#1496]
**Is your feature request related to a problem? Please describe.**
suggestion: would be great to see some kind of a summary here (# of samples, # success/failure ratio)

**Additional context**
![image](https://github.com/user-attachments/assets/35a672c1-2c84-4524-b020-c9a61ca306f0)

**Comment by odbol** [comment]
Yes! I'd love to see some total scores, maybe even a graph of the eval scores from every run. That way we can actually compare how different eval runs (with different prompts etc) perform.

**Comment by ssbushi** [comment]
I'm tracking this in #2252, closing this issue.
